src/go/k8s/Makefile

```diff
@@ -0,0 +1,101 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= gcr.io/vectorized/controller:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
```

Comment:
do we need to be compatible with that? I would be fine with compatibility with the currently supported k8s which is 1.20 - 3 versions, so 1.17?

Reply:
As far as I know:
- EKS supports version from 1.15 to 1.18
- GKE supports version from 1.15.12 to 1.17.15
- Azure kubernetes service supports version from 1.16 to 1.19

Those versions are requirements for `redpanda` operator.

Reply:
If you spin up a cluster in GKE, it by default gives you a `v1.16.15-gke.6000` cluster.

Reply:
I will leave that comment in `Makefile`. In the next PR, I will create github action that exercise mentioned versions of the kuberntes agains `Kind` clusters.